molo/core/cookiecutter/scaffold/{{cookiecutter.directory}}/{{cookiecutter.app_name}}/templates/wagtailadmin/home/upgrade_notification.html

```diff
@@ -0,0 +1,10 @@
+{% raw %}
```

Comment:
@miltontony these templates are in the cookiecutters so they need the `{% raw %}` tag for scaffolding